src/lib.rs

```diff
@@ -148,6 +157,39 @@ impl Game {
             if inputs.right {
                 player.x += self.speed * time_steps_passed
             }
+
+            if !inputs.q && self.last_input.q {
+                kill_position = Some((player.x, player.y));
+            }
+        }
+
+        match kill_position {
+            Some(position) => self.kill_player_near(position)?,
+            None => return Ok(()),
+        }
+
+        Ok(())
+    }
+
+    fn kill_player_near(&mut self, position: (f64, f64)) -> Result<(), &'static str> {
+        let local_player_color = match &self.local_player_color {
+            None => return Ok(()), // not controlling anything
+            Some(c) => *c,
+        };
+
+        for player in self.players.iter_mut() {
```

Comment:
Yeah, a way to iterate over other players would be nice.

Writing iterators in Rust today isn't fun, I really want a `yield` construct